pallets/ah-migrator/src/lib.rs

```diff
@@ -462,11 +467,16 @@ pub mod pallet {
 		#[pallet::call_index(0)]
 		#[pallet::weight({
 			let mut total = Weight::zero();
+			let weight_of = |account: &RcAccountFor<T>| if account.is_liquid() {
```

Comment:
This is doing a free storage access, right?

Reply:
no storage access here, only message data needed